packages/@aws-cdk/aws-lambda/lib/scalable-function-attribute.ts

```diff
@@ -0,0 +1,81 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Interface for scalable attributes
+ */
+export interface IScalableFunctionAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level. The utilization is tracked by the
+   * LambdaProvisionedConcurrencyUtilization metric, emitted by lambda. See:
+   * https://docs.aws.amazon.com/lambda/latest/dg/monitoring-metrics.html#monitoring-metrics-concurrency
+   *
+   * Allowed values: 0.1 - 0.9.
+   */
+  scaleOnUtilization(props: UtilizationScalingOptions): void;
+  /**
+   * Scale out or in based on schedule.
+   */
+  scaleOnSchedule(id: string, actions: appscaling.ScalingSchedule): void;
+}
+
+/**
+ * Properties for enabling Lambda utilization tracking
+ */
+export interface UtilizationScalingOptions extends appscaling.BaseTargetTrackingProps {
+  /**
+   * Target utilization value for the attribute. For example, .5 indicates that 50 percent of allocated provisioned concurrency is in use.
+   *
+   * Allowed values: 0.1 - 0.9.
+   */
+  readonly targetUtilizationValue: number;
```

Comment:
The word "value" does not add any value ;-) here. Also, feels like it's `utilizationTarget` (reads like english):

```suggestion
  readonly utilizationTarget: number;
```